### ISSUE TITLE
Firevim Support

### ISSUE THREAD

**Issue by joshmedeski** [pwntester/octo.nvim#556]
It would be amazing to integrate octo.nvim with [firevim](https://github.com/glacambre/firenvim?tab=readme-ov-file). I'm experimenting with a custom firevim config and would like access to Octo.nvim's features.

Firevim names the buffer based off the URL, so perhaps we could parse the buffer to detect which repo to connect to.

**Comment by pwntester** [comment]
Hey 👋 

Can you elaborate more on the use case you have in mind? Do you want to use octo when writing GitHub comments/reviews from the WebUI?

**Comment by joshmedeski** [comment]
Yes exactly, there is a lot of potential here since there are so many pages on GitHub that can be mapped to Octo. But things like autocomplete for tagging issues and PRs come to mind as a a good first item.
